src/Components/Blazor/Build/src/Microsoft.AspNetCore.Blazor.Build.csproj

```diff
@@ -1,9 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetName>Microsoft.AspNetCore.Blazor.Build.Tasks</TargetName>
```

Comment:
Is it OK that the assembly name and project/dir name don't match? I mean, I understand it works technically, but this might confuse people as it doesn't match our normal conventions. Could we rename the dir and project to `Microsoft.AspNetCore.Blazor.Build.Tasks`?

Reply:
We do something similar with the Razor.SDK. We want to keep the package name `Microsoft.AspNetCore.Blazor.Build`, but by convention most task assemblies are named 'Something.Tasks`. The task assembly is generally not referenced by any code so it might not be incredibly confusing. Let me know if you feel strongly and we can figure out other options to organize this.

Reply:
I'm fine with whatever here